Added OHS implementation

bashir2 opened the pull request:
The format of the `test-result.json` file is slightly different, e.g., it has a `result.reason` field for tests that we skip. I'll update the documentation in a follow-up PR.
This PR also removes the duplicated `implementations.json` to fix #237.

johngrimes reviewed (changes requested):
You've got the test result directly within the test object:

```json
{
  "name": "basic attribute",
  "passed": true
}
```

It needs to be nested inside a "result" field, like this:

```json
{
  "name": "basic attribute",
  "result": {
    "passed": true
  }
}
```

johngrimes commented:
If you want to preview what the test report will look like, you can run the app inside the `test_report` up locally.

bashir2 commented:
> If you want to preview what the test report will look like, you can run the app inside the `test_report` up locally.

Thanks for the tip; I ran the app locally and also updated some of the instructions. As a side effect, I also fixed #237 and updated documents accordingly.

bashir2 commented:
> You've got the test result directly within the test object:
> 
> ```json
> {
>   "name": "basic attribute",
>   "passed": true
> }
> ```
> 
> It needs to be nested inside a "result" field, like this:
> 
> ```json
> {
>   "name": "basic attribute",
>   "result": {
>     "passed": true
>   }
> }
> ```

This is now fixed in the [latest version of our test report](https://raw.githubusercontent.com/google/fhir-data-pipes/master/pipelines/common/src/test/resources/sql-on-fhir-v2-test-result.json).

bashir2 commented:
Thanks @johngrimes for merging this. Looking at [the implementations page](https://fhir.github.io/sql-on-fhir-v2/#impls) it cannot fetch Pathling's test report. It seems it throws a CORS error; can you please take a look?
